index.html

```diff
@@ -1,21 +1,112 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Karma</title>
+    <title>Karma Abbas</title>
     <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
     <!-- Add a link to your CSS file here (use the line above to guide you) -->
     <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
 </head>
+
 <body>
+    <!-- navigation bar -->
+    <header>
+        <nav class="navbar">
+            <img class="logo" src="img/karma-logo.svg" alt="Karma Logo">
+            <ul class="links">
+                <li>
+                    <a href="#"><span class="dark">Meet Karma</span></a>
+                </li>
+                <li>
+                    <a href="#">How it Works</a>
+                </li>
+                <li>
+                    <a href="#">Store</a>
+                </li>
+                <li>
+                    <a href="#">Blog</a>
+                </li>
+                <li>
+                    <a href="#">Help</a>
+                </li>
+                <li>
+                    <a href="#">Login</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+
+    <!-- SHOWCASE -->
+    <section class="showcase">
+        <h2>
+            Introducing Karma
+        </h2>
+        <h4>
+            Bring WiFi with you, everywhere you go.
+        </h4>
+
+        <button class="button" type="button">
+            Learn More
+        </button>
 
+    </section>
+
+    <!-- SEGMENTS -->
+    <section class="seg">
```

Comment:
Be careful with shortened class names - I wasn't sure what 'seg' meant when I first read it.